solidity-plugin/src/main/java/org/sonarsource/solidity/rules/SolidityRulesDefinition.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Example Plugin for SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.solidity.rules;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.api.server.rule.RulesDefinitionXmlLoader;
+import org.sonarsource.solidity.languages.Solidity;
+
+public final class SolidityRulesDefinition implements RulesDefinition {
+
+  private static final String PATH_TO_RULES_XML = "/solidity/solidity-rules.xml";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=11&issues=AWPzDwcS3e-qcckj23qx&open=AWPzDwcS3e-qcckj23qx)

[ISSUE_KEY]: AWPzDwcS3e-qcckj23qx